src/gui/maptools/qgsmaptoolcapture.cpp

```diff
@@ -674,6 +671,10 @@ int QgsMapToolCapture::fetchLayerPoint( const QgsPointLocator::Match &match, Qgs
           layerPoint.dropMValue();
         }
 
+        if ( sourceLayer->crs() != vlayer->crs() )
+        {
+          layerPoint = toLayerCoordinates( vlayer, layerPoint );
+        }
```

Comment:
Doesn't this extra transformation cause #60927 & #60868 ?

Reply:
Yes, but fixes #59792 ... When I found a for one, it's adding regression to another one...
Was AFK last days, and working on it since this morning, and sometimes I would expect that projection doesn't exist...
Anyway, I found other strange logic in QgsMapToolCapture. I hope to find a solution soon...